gapis/api/graph_visualization.go

```diff
@@ -0,0 +1,35 @@
+// Copyright (C) 2018 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package api
+
+const (
+	MAXIMUM_LEVEL_IN_HIERARCHY = 10
+)
+
+type Hierarchy struct {
+	LevelId      [MAXIMUM_LEVEL_IN_HIERARCHY]int
```

Comment:
I changed this in slice, also I added functions setValue and GetValue for this slice, because its length could increment dynamically.

Reply:
I added functions to manage the Hierarchy in a better and simpler way. In this commit for the trace "multithreading" is obtained unexpected results because multiple threads running , but this problem is addressed in a later commit.